setup/ibus-setup.desktop

```diff
@@ -7,3 +7,6 @@ Terminal=false
 Type=Application
 StartupNotify=true
 Categories=Settings;
+# GNOME uses its own "input source" settings instead
+NotShowIn=GNOME;Unity;
```

Comment:
Pretty sure this should also include `GNOME-Flashback` in the list:
```
NotShowIn=GNOME;GNOME-Flashback;Unity;
```

Reply:
I don't see GNOME-Flashback in Fedora but I guess you could modify DesktopNames in the GNOME desktop file instead of the IBus desktop file.
```
DesktopNames=GNOME-Flashback;GNOME;
```